tabular/src/autogluon/tabular/__init__.py

```diff
@@ -10,4 +9,4 @@
 
 from .predictor import TabularPredictor
 
-logging.basicConfig(format='%(message)s')  # just print message in logs
+_add_stream_handler()
```

Comment:
Technically this is true. However, there are modules that didn't play around with the root logger like tabular and time series does. I'm afraid this will have side effect on those module, so I stick to what the old logic is.